blocks/simple-cards/simple-cards.css

```diff
@@ -0,0 +1,147 @@
+.simple-cards {
+    margin: 0 auto;
+    max-width: 100%;
+    padding: 0 20px;
+
+    & img {
+        width: 100%;
+        max-height: 193px;
+        aspect-ratio: 3 / 2;
+    }
+
+    .simple-cards-grid-activities {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(3, auto);
+        column-gap: 12px;
+        max-width: 100%;
+
+        & picture {
+            filter: brightness(0.7);
+        }
+    }
+
+    .simple-cards-grid-regions {
+        display: block;
+    }
+
+    .card-figure {
+        margin: 0;
+        overflow: hidden;
+        position: relative;
+        padding: 5px;
+        border: 1px solid #dadada;
+        background-color: #fff;
+        max-height: inherit;
+        box-sizing: border-box;
+    }
+
+    .card-image > picture > img {
+        display: block;
+        width: 100%;
+        height: 193px;
+        object-fit: cover;
+    }
+
+    .card-image-activities > picture > img {
+        aspect-ratio: 1;
+        height: auto;
+        display: block;
+        object-fit: cover;
+    }
+
+    .card-caption {
+        font-family: Franklin, sans-serif;
+        font-weight: 700;
+        font-style: italic;
+        text-shadow: 1px 2px 2px #666;
+        color: #fff;
+        z-index: 1;
+        width: 100%;
+        font-size: 1.5em;
+        white-space: nowrap;
+    }
+
+    .card-caption-with-icon {
+        font-family: Franklin, sans-serif;
+        font-weight: 700;
+        font-style: italic;
+        text-shadow: 1px 2px 2px #666;
+        color: #fff;
+        z-index: 1;
+        width: 100%;
+        font-size: 1em;
+        white-space: wrap;
+        text-align: center;
+    }
+
+    .card-figure-overlay {
+        position: absolute;
+        display: flex;
+        flex-direction: column;
+        z-index: 999;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        align-items: center;
+    }
+
+    .card-icon {
+        display: block;
+        width: 45px;
+        height: 45px;
+    }
+
+    & > ul > li {
+        position: relative;
+        height: calc(100% - 10px);
+        margin-bottom: 12px;
+    }
+}
+
+@media (width >= 465px) {
+    .card-image-activities > picture > img {
+        aspect-ratio: 1;
+        height: auto;
+    }
+}
```

Comment:
This isn't necessary - to reason: it's not as qualified as the declaration at line 49. And it's repeating what's in that declaration, so it can just be removed.